### ISSUE TITLE
The Resource pack doesn't work.

### ISSUE THREAD

**Issue by Seagkpeagl** [Wynncraft/Issues#6976]
Once I join a server and pick my class I load into the last spot I was in but with no resource pack and tells me to go to this link https://forums.wynncraft.com/threads/cant-play-wynncraft-because-of-the-pack-heres-how-to-fix.98023/ I've done all of the solutions besides (none of them worked) the "%appdata% folder" but I cant find that folder on my computer . Someone please help me.

**Comment by Numbermaniac** [comment]
> the "%appdata% folder" but I cant find that folder on my computer.

Open File Explorer, and in the address bar, replace everything with %appdata%, then hit enter.

**Comment by Seagkpeagl** [comment]
I'm using a mac

**Comment by Numbermaniac** [comment]
Oh right. @Seagkpeagl open up Finder, hit Command-Shift-G, and in the box that opens, type in `~/Library/Application Support/Minecraft` and hit enter. That'll take you to the right folder.

**Comment by Seagkpeagl** [comment]
It takes me too all of these folders and documents. https://gyazo.com/468f40a22ff6e5926bac9db2fa60d07d
I assume the folder is "server-resourse-packs" and that contains 10 documents with random number and letters. https://gyazo.com/38d703bf7e58b995ead0c3d76b105caa

**Comment by Numbermaniac** [comment]
Yeah, that's where they are stored, but for some reason it does use rather cryptic naming. I'd suggest you delete them all, and the next time you launch Wynncraft it should download it again.

**Comment by Seagkpeagl** [comment]
I deleted all 10 of documents and emptied my trash bin. I then launched minecraft and joined the server. I still don't have the resource pack. Any other suggestions?

**Comment by Numbermaniac** [comment]
Do you have Server Resource Packs on? It was in the page you linked earlier.

**Comment by Seagkpeagl** [comment]
Its always been on, and still nothing.

**Comment by Moe-Ronickah** [comment]
Try [this page](https://github.com/Wynncraft/Issues) , follow the instructions for downloading the latest resource pack, then follow the instructions above and put in the appropriate folder.  Let us know what happens.
